peps/pep-9999.rst

```diff
@@ -14,41 +15,38 @@ Post-History: 02-Jun-2019,
 Abstract
 ========
 
-This PEP proposes a new module for number-theoretical, combinatorial and other
-integer-valued functions defined for integer arguments, like
-:external+py3.14:func:`math.gcd` or :external+py3.14:func:`math.isqrt`.
+This PEP proposes a new module for mathematical functions related to number
+theory and combinatorics.
```

Comment:
Could you argue why this is better?

Reply:
According to Wikipedia:
* combinatorics and number theory are different branches of mathematics, and
* combinations (binomial coefficient) and permutations are part of combinatorics.

Therefore, I think our desired scope should be "number theory and combinatorics".

I also think that abstracts should be as concise as possible with details presented in papers/proposals.  So I prefer one sentence without examples.